great_expectations/data_context/util.py

```diff
@@ -128,3 +130,17 @@ def safe_mmkdir(directory, exist_ok=True):
     except OSError as e:
         if e.errno != errno.EEXIST:
             raise
+
+
+def find_context_root_dir():
```

Comment:
Any particular reason to put this in `util` rather than in the `DataContext` itself?